pages/api/helpjuice/[...path].page.ts

```diff
@@ -0,0 +1,48 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import fetch from 'node-fetch';
+
+export const buildUrlPath = (originalPath: string | string[] | undefined) => {
+  if (!originalPath) {
+    return originalPath;
+  }
+  let pathArray;
+  if (typeof originalPath === 'string') {
+    pathArray = originalPath.split(',');
+  } else {
+    pathArray = originalPath;
+  }
+  return pathArray.join('/');
+};
```

Comment:
You might be able to simplify this to the below.

- Using the optional variable means you don't need to specify undefined.
- Then we can simplify the bottom half

```
export const buildUrlPath = (originalPath?: string | string[]) => {
  if (!originalPath) {
    return originalPath;
  }
  return Array.isArray(originalPath)
    ? originalPath.join('/')
    : originalPath.split(',').join('/');
};
```